src/pythermalcomfort/jos3_functions/construction.py

```diff
@@ -38,6 +38,51 @@
     ]
 )
 
+# Anthropomorphic data for a standard body
+default_height = 1.72
```

Comment:
Great, I like this change since you only define these default values once in the code. On the other hand, should not we add them to a dataclass which contains all the defaults so it is easier to keep track of them and we do not have to prepend each variable name with the term default? Something like the following

```
@dataclass
class Defaults:
     height: float = 1.72
     weight: float = 74.43
```

Reply:
OK. Created a default data class with anthropomorphic data for a default body in `construction.py`.